examples/pnet/libp2p-node.js

```diff
@@ -13,7 +13,7 @@ const Protector = require('libp2p/src/pnet')
  * @param {Buffer} swarmKey
```

Comment:
There is one more reference to `swarmKeyPath` in the docs above. Can you change it?

Reply:
oops, fixed